templates/prometheusremotewriterelabelconfig.go

```diff
@@ -0,0 +1,43 @@
+package templates
+
+import (
+	"strings"
+
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+)
+
+var metrics = []string{
+	"CephMdsMissingReplicas",
+	"CephMgrIsAbsent",
+	"CephMgrIsMissingReplicas",
+	"CephNodeDown",
+	"CephClusterErrorState",
+	"CephClusterWarningState",
+	"CephOSDVersionMismatch",
+	"CephMonVersionMismatch",
+	"CephOSDFlapping",
+	"CephOSDDiskNotResponding",
+	"CephOSDDiskUnavailable",
+	"CephDataRecoveryTakingTooLong",
+	"CephPGRepairTakingTooLong",
+	"CephMonQuorumAtRisk",
+	"CephMonQuorumLost",
+	"job:ceph_versions_running:count",
+	"job:ceph_pools_iops_bytes:total",
+	"job:ceph_pools_iops:total",
+	"job:kube_pv:count",
+	"job:ceph_osd_metadata:count",
+	"ceph_health_status",
+	"ceph_cluster_total_used_raw_bytes",
+	"ceph_cluster_total_bytes",
+	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
+	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
+}
+
+var PrometheusRemoteWriteRelabelTemplate = []promv1.RelabelConfig{
+	{
+		SourceLabels: []string{"__name__"},
+		Regex:        strings.Join(metrics[:], "|"),
+		Action:       "keep",
+	},
+}
```

Comment:
This entire thing need to be rolled into the prom template as it is a static part of that template

Reply:
Done